src/helpers/visitStatus.js

```diff
@@ -0,0 +1,6 @@
+// Defined the set of statuses that a visit can have (status column in scheduled_calls_table)
+
+export const SCHEDULED = "scheduled"; // Default - initial state
+export const ARCHIVED = "archived"; // The visit's parent ward has been archived
+export const CANCELLED = "cancelled"; // The visit has been explicitly cancelled
+export const ADMIN = "complete"; // The visit has occurred
```

Comment:
```suggestion
export const COMPLETE = "complete"; // The visit has occurred
```

Looking at `db/scripts/cleanup_scheduled_calls.js`, it looks like we want this instead?🕵️‍♀️  Maybe it should be past tense like the others?

Reply:
That's a great spot and on reflection also explains why the db seeding was failing as I was referencing `COMPLETED` in that script :facepalm: 
I'll fix it...

Reply:
Actually, it doesn't explain the seeding issue as that was `status = SCHEDULED`...

Reply:
The name is fixed now...